inference-engine/ie_bridges/python/tests/test_InferRequest.py

```diff
@@ -452,3 +452,35 @@ def test_blob_setter(device):
     request.infer()
     res_2 = np.sort(request.output_blobs['fc_out'].buffer)
     assert np.allclose(res_1, res_2, atol=1e-2, rtol=1e-2)
+
+
+def test_getting_preprocess(device):
+    ie_core = ie.IECore()
+    net = ie_core.read_network(test_net_xml, test_net_bin)
+    exec_net = ie_core.load_network(network=net, device_name=device, num_requests=1)
+    request = exec_net.requests[0]
+    preprocess_info = request.preprocess_info["data"]
+    assert isinstance(preprocess_info, ie.PreProcessInfo)
+    assert preprocess_info.mean_variant == ie.MeanVariant.NONE
+
+
+def test_blob_setter_with_preprocess(device):
+    ie_core = ie.IECore()
+    net = ie_core.read_network(test_net_xml, test_net_bin)
+    exec_net_1 = ie_core.load_network(network=net, device_name=device, num_requests=1)
+
+    net.input_info['data'].layout = "NHWC"
+    exec_net_2 = ie_core.load_network(network=net, device_name=device, num_requests=1)
+
+    img = read_image()
+    res_1 = np.sort(exec_net_1.infer({"data": img})['fc_out'])
+
+    img = np.transpose(img, axes=(0, 2, 3, 1)).astype(np.float32)
+    tensor_desc = ie.TensorDesc("FP32", [1, 3, 32, 32], "NHWC")
+    img_blob = ie.Blob(tensor_desc, img)
+    preprocess_info = ie.PreProcessInfo()
```

Comment:
is it possible to create PreProcessInfo with some initializaed fields via constructor (like 
```python
ie.PreProcessInfo(resize_algorithm=ie.ResizeAlgorithm.BILINEAR)
````
)
or only way is create default and change proprerties?
```python
preprocess_info = ie.PreProcessInfo()
preprocess_info.resize_algorithm = ie.ResizeAlgorithm.BILINEAR
```